README.md

```diff
@@ -4,33 +4,47 @@
 
 1. `git add .`
 2. `git commit -m "my work"`
-3. `git checkout week2`
+3. `git checkout master`
```

Comment:
Why checkout `master` instead of `week2`?

Reply:
I am structuring this so that the read me will be used in class and students who want to get caught up can follow those directions.  Any student who was in class the week before will just build off of what they already coded.  If they pull week2 today they will get the completed code and might not be as involved in the process.

Reply:
Makes sense, thanks for the info

Reply:
Normally week2 didn't have completed code...just the starting point for week 1.  Otherwise if they miss a session then they can't get the code for the next session they are working on.  so really it is suppose to be 
master = week 1 start
week2 = week 1 end
week3 = week 2 end
week4 = week 3 end.
weekN = week N-1 end